tests/test-war-smoke/pom.xml

```diff
@@ -228,19 +154,26 @@
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-failsafe-plugin</artifactId>
-            <version>2.19.1</version>
-            <configuration>
-              <systemPropertyVariables>
-                <app.deploy.project>${app.deploy.project}</app.deploy.project>
-                <app.deploy.version>${maven.build.timestamp}</app.deploy.version>
-              </systemPropertyVariables>
-            </configuration>
             <executions>
               <execution>
+                <id>remote-integration-test</id>
                 <phase>integration-test</phase>
+                <goals>
+                  <goal>integration-test</goal>
+                  <goal>verify</goal>
+                </goals>
+                <configuration>
+                  <skipTests>false</skipTests>
+                  <systemPropertyVariables>
+                    <test.mode>remote</test.mode>
+                    <app.deploy.project>${app.deploy.project}</app.deploy.project>
+                    <app.deploy.version>${maven.build.timestamp}</app.deploy.version>

```

Comment:
For the sake of consistency, can we use `${docker.tag.long}` here for the version?